XbSymbolDatabase.h

```diff
@@ -482,6 +482,12 @@ uint32_t XbSymbolLibrayToFlag(const char* library_name);
 /// <returns>Return total count of errors.</returns>
 unsigned int XbSymbolDataBaseTestOOVPAs();
 
+/// <summary>
+/// Set to true will perform full range of OOVPAs registered. Or stop at title's revision detected.
```

Comment:
What does "stop at" refer to?
Do you mean "Only scan title revision"?

Is this setting per-library, or what's a "title's revision" (= a generic one across all libs?)

Reply:
Xbe's revision to be exact. The scan itself collect a list of each library's revision then determine which one is the highest. Then use the highest revision to end the scan process in order to avoid false positive. Since there are couple/some symbols can only use a offset value which does go up then back down to same value. After all, those symbols do have complete asm codes being identical except for the offset.

Reply:
Okay.

Minor notes which affect the code comment:

> Xbe's revision to be exact.

There's no such field in the XBE headers. I assume you meant assembled version info from the libraries, as explained afterwards.

> The scan itself collect a list of each library's revision

I assume you mean "library build numbers" - this is important because "revision" refers to something else. The Alpha XDKs had a binary versioned with ` 3146.3 ` for example, which is build 3146, but QFE / Revision 3. Also see <https://docs.microsoft.com/en-us/windows/desktop/api/shlwapi/nf-shlwapi-makedllverull>